pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -81,13 +81,11 @@ func (op *overcommitPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 
 	// calculate idle resources of total cluster, overcommit resources included
-	total := api.EmptyResource()
 	used := api.EmptyResource()
 	for _, node := range ssn.Nodes {
-		total.Add(node.Allocatable)
 		used.Add(node.Used)
 	}
-	op.idleResource = total.Clone().Multi(op.overCommitFactor).Sub(used)
+	op.idleResource = ssn.TotalResource.Clone().Multi(op.overCommitFactor).Sub(used)
```

Comment:
It is suggested that the definition of TotalResource can be added in the overcommitPlugin structure, and assigned by ssn.TotalResource at the beginning of OnSessionOpen, see similar implementations of drf and proportion

Reply:
It sounds reasonale. I will update it.
By the way, is it need to also put the cluster's used at ssn so that all plugins don't need to calculate once again when use it.